doc/topology.md

```diff
@@ -110,3 +110,55 @@ and disabled.
 After that, all instance files
 (configuration file, socket and working directory)
 are removed on [`cleanup_expelled`](/doc/scenario.md#cleanup_expelled) step.
+
+## Advertise URIs changing
+
+It is possible to change the advertise URIs of the servers. However, this should be done carefully.
+
+The main problem that arises when changing URIs: after restart, the instance tries
+to connect to all its replicas and remains in the ConnectingFullmesh state until it succeeds.
+If it can’t (due to replica URI unavailability or for any other reason) – it’s stuck forever.
+
+To solve the problem, you should set the `replication_connect_quorum` option to zero.
```

Comment:
Please, describe that this change is performed on `configure_instance` step and doesn't require instance restart.